meson_options.txt

```diff
@@ -10,8 +10,8 @@ option('introspection', type: 'boolean',
 option('gcc_vector', type: 'boolean',
        value: true,
        description: 'Enable GCC vector fast paths (requires GCC)')
-option('sse2', type: 'boolean',
-       value: true,
+option('sse2', type: 'feature',
```

Comment:
I don't want this to turn into a feature, sorry.

Reply:
So let's keep it a boolean.

My original thinking was to make it possible to enable SSE2 for 32-bit "forcefully" with `-Dsse2=enabled`.

Reply:
I'd rather not allow that; if we know that SSE types on 32bit architectures are problematic—especially when used by other libraries that don't have alignment guarantees—then I'd rather not give out footguns to people.